### PR TITLE
Added BStack to name

### PR DESCRIPTION
**Pull request by agarneha1331**

### CODE REVIEW

File: tests/acceptance.suite.yml [hunk 1]
```diff
@@ -19,15 +19,15 @@ modules:
         'browserstack.user': 'BROWSERSTACK_USERNAME'
         'browserstack.key' : 'BROWSERSTACK_ACCESS_KEY'
         'browserstack.debug': true
-        'build': 'codeception-browserstack'
+        'build': 'browserstack build'
```
Comment (Ankit098) [inline tests/acceptance.suite.yml]:
```suggestion
        'build': 'browserstack build 1'
```
i think this is used in other repos..

File: tests/acceptance.suite.yml [hunk 2]
```diff
@@ -36,36 +36,36 @@ env:
...
+            'name': 'BStack parallel test'
 
   parallel_3:
     modules:
       config:
         \BrowserStackWebDriver:
           browser: 'internet explorer'
           capabilities:
-            'name': 'parallel_test'
+            'name': 'BStack parallel test'
```
Comment (Ankit098) [inline tests/acceptance.suite.yml]: verify this once.. IE i don't think we have in other repos can be removed (can keep the 3 combinations above chrome ff safari)

Comment (agarneha1331) [reply tests/acceptance.suite.yml]: removed this for now

File: tests/acceptance.suite.yml (hunk 1 above)
Comment (agarneha1331) [reply tests/acceptance.suite.yml]: done

**Review by karanshah-browserstack — APPROVED** [review]

**Review by francisf — APPROVED** [review]

**Pull request merged**